components/prism-graphql.js

```diff
@@ -18,7 +18,17 @@ Prism.languages.graphql = {
 	},
 	'number': /(?:\B-|\b)\d+(?:\.\d+)?(?:e[+-]?\d+)?\b/i,
 	'boolean': /\b(?:true|false)\b/,
+	'variable-input': {
+		pattern: /(mutation.*)\$[\w_]+/i,
```

Comment:
1. Keywords are case-sensitive, so no `i` flag.
2. We need a word boundary before `mutation`.
3. `[\w_]`: `_` is included by `\w`

```suggestion
		pattern: /(\bmutation.*)\$\w+/,
```